news/1681.bugfix

```diff
@@ -0,0 +1 @@
+Fix the --cfg=job --resolve flags so that the ${hydra:...} resolver now prints properly
```

Comment:
This fixes --help as well?

Reply:
Yes. I updated the news fragment in https://github.com/facebookresearch/hydra/pull/1696/commits/751653f665956141f43453eb30814cd72513d694.